raspi/soundserver/templates/index.html

```diff
@@ -6,78 +6,255 @@
-          		</button>
-          </fieldset>
-          <button onclick="window.location.href = '/set_password';" class="btn btn-warning m-2">
-            Set or Change Password
-            <i class="fa fa-key"></i>
-          </button>
-
-        </div>
-    </div>
+          
+      </div>
+           
     <script>
-      function control(action){
-        console.log(action)
-        var http = new XMLHttpRequest();
-        var url = '/'+action;
-        http.open('POST', url, true);
-        http.send();
-      }
-      function SetVolume(val){
-        console.log(val)
-        var http = new XMLHttpRequest();
-        var url = '/volume?val='+val;
-        http.open('POST', url, true);
-        http.send();
-      }
-    </script>
-  </body>
-</html>
+          function control(action){
+            console.log(action)
+            var http = new XMLHttpRequest();
+            var url = '/'+action;
+            http.open('POST', url, true);
+            http.send();
+          }
+          function reset(action){
+            console.log(action)
+            var http = new XMLHttpRequest();
+            var url = '/reset_smart_speaker/'+action;
+            http.open('POST', url, true);
+            http.send();
+          }
+          function SetVolume(val){
+            console.log(val)
+            var http = new XMLHttpRequest();
+            var url = '/volume?val='+val;
+            http.open('POST', url, true);
+            http.send();
+          }
+         
+          function getOfflineSong(){
+            console.log("inside getOfflineSong")
+            var url = '/getOfflineSong';
+    
+             fetch(url).then(function(response) {
+              return response.json();
+            }).then(function(data) {
+             var mod =  document.createElement("div")
+             mod.setAttribute('class',"list")
+              data.status.forEach(song => {
+                var btn = document.createElement("button");
+                var icon = document.createElement("i")
+                var name = document.createTextNode(song.name);
+                icon.className = "fa fa-music"
+                btn.value=song.name
+                btn.appendChild(icon)
+                btn.appendChild(name)
+                btn.className= "btn btn-outline-primary m-2"
+                btn.setAttribute('onClick','playOfflineSong(this)')
+                mod.appendChild(btn)
+              });
+              document.getElementById('layout').appendChild(mod)
+            }).catch(function() {
+              console.log("Error");
+              return null;
+            });
+          }
+          function playOfflineSong(e){
+            var song =  e.value
+            console.log("Song: "+song)
+            var url = '/playOfflineSong/'+song;
+            console.log("url="+url)
+            var http = new XMLHttpRequest();
+            http.open('PUT', url, true);
+            http.send();
+          //   fetch(url, {
```

Comment:
@aker99 Kindly remove unused code and finalise this PR.

Reply:
changed in the new commit